Utils/Dataflow/091_datasetsRucio/datasets_processing.py

```diff
@@ -40,16 +40,39 @@
     sys.exit(1)
 
 rucio_client = None
-DS_TYPE = 'output'
+
+OUTPUT = 'o'
+INPUT = 'i'
+
+META_FIELDS = {
+    OUTPUT: {'bytes': 'bytes', 'events': 'events', 'deleted': 'deleted'},
+    INPUT: {'bytes': 'input_bytes', 'deleted': 'primary_input_deleted'}
+}
+
+SRC_FIELD = {
+    OUTPUT: 'output',
+    INPUT: 'primary_input'
+}
 
 
 def main(argv):
     """ Program body. """
     stage = pyDKB.dataflow.stage.JSONProcessorStage()
+    stage.add_argument('-t', '--dataset-type', action='store', type=str,
+                       help=u'Type of datasets to work with: (i)nput'
+                             ' or (o)utput',
+                       nargs='?',
+                       default=OUTPUT,
+                       choices=[INPUT, OUTPUT],
```

Comment:
As I understood, to set dataset_type we should do something like:

{{{
$ ./datasets_processing.py -t 'i'
}}}

Am I right?

Reply:
Right. And without quotes it also must work properly :)